file_organizer_v2/examples/image_dedup_example.py

```
@@ -0,0 +1,240 @@
+        directory,
+        recursive=True,
+        progress_callback=progress_callback
+    )
+
+    print()
+    print(f"Found {len(duplicates)} groups of duplicate images")
+    print()
+
+    # Display results
+    if duplicates:
+        for idx, (hash_key, images) in enumerate(duplicates.items(), 1):
```

Ruff (0.14.13)

Loop control variable `hash_key` not used within loop body

Rename unused `hash_key` to `_hash_key`

(B007)